packages/models-library/src/models_library/api_schemas_dynamic_sidecar/containers.py

```diff
@@ -16,3 +16,5 @@ class ActivityInfo(BaseModel):
 
 
 ActivityInfoOrNone: TypeAlias = ActivityInfo | None
+
+DockerComposeYamlStr: TypeAlias = str
```

Comment:
Please have a look at: https://pydantic-yaml.readthedocs.io/en/latest/

Reply:
Compose spec is not an issue it is validated via the docker-cli. This is just a convenient format in which is stored before it is used for various commands. it's like an artefact that is no longer touched